src/XIVLauncher.Core/LauncherApp.cs

```diff
@@ -265,14 +265,15 @@ public override void Draw()
         ImGui.SetNextWindowPos(new Vector2(0, 0));
         ImGui.SetNextWindowSize(ImGuiHelpers.ViewportSize);
 
+        ImGui.PushStyleColor(ImGuiCol.WindowBg, ImGuiColors.BlueShade0);
```

Comment:
`ImGui.PushStyleColor(ImGuiCol.WindowBg, ImGuiColors.BlueShade0);`

This is the bit of code which is causing the window to be colored differently. I'm not sure *why*, however, since it doesn't actually do anything here. Comment it out (and the corresponding Pop), and you get exactly the same color.

Oddly, if you comment out the equivalent code line in the main branch, the window has an almost black background. Furthermore, you can move the PushStyleColor line almost anywhere in the LauncherApp file and it'll still work, but delete it completely or comment it out, and it has the black background.

Reply:
This isn't the main driver for the color issue. found out where it is coming from and it is `private Vector4 clearColor = new(0.45f, 0.55f, 0.60f, 1.00f);` in ImGuiBindings